File: lib/core/strings.dart
/// Uygulamadaki tüm kullanıcıya görünen metinler burada toplanır.
///
/// Kural: widget dosyalarının içine string literal yazılmaz. Yeni bir metin
/// gerektiğinde önce buraya `static const` olarak eklenir, sonra kullanılır.
class Strings {
  const Strings._();

  // ---------------------------------------------------------------- Genel
  static const String appTitle = 'Filo Takip';
  static const String loading = 'Yükleniyor…';
  static const String retry = 'Tekrar dene';
  static const String cancel = 'Vazgeç';
  static const String close = 'Kapat';
  static const String ok = 'Tamam';

  // ------------------------------------------------------- Alt gezinme
  static const String tabMap = 'Harita';
  static const String tabRegistration = 'Kayıt';
  static const String tabAdmin = 'Yönetim';
  static const String tabSettings = 'Ayarlar';

  // ------------------------------------------------------- Açılış / oturum
  static const String startupConnecting = 'Sunucuya bağlanılıyor…';
  static const String startupSigningIn = 'Oturum açılıyor…';
  static const String startupFailedTitle = 'Uygulama başlatılamadı';
  static const String startupFirebaseFailed =
      'Firebase bağlantısı kurulamadı. İnternet bağlantınızı kontrol edin. '
      'Sorun sürerse google-services.json dosyasının projeye eklendiğinden '
      'emin olun.';
  static const String startupSignInFailed =
      'Oturum açılamadı. Firebase Console üzerinde anonim giriş sağlayıcısının '
      'etkin olduğundan emin olun.';

  // ---------------------------------------------------------- Simülasyon
  static const String simulationBadge = 'SİMÜLASYON';

  /// Köşe şeridine sığması gerektiği için kısa tutuldu.
  static const String simulationRibbon = 'DEMO';
  static const String simulationNotice =
      'Uygulama şu anda sahte verilerle çalışıyor. Hiçbir veri sunucuya '
      'yazılmıyor ve uygulama kapanınca değişiklikler kaybolur.';
  static const String simulationAdminHint =
      'Simülasyon yöneticisi: yonetici@ornek.com / 123456';

  // ------------------------------------------------------------- Hatalar
  static const String errorInvalidCredentials = 'E-posta veya şifre hatalı.';
  static const String errorNetwork =
      'Ağ bağlantısı kurulamadı. İnternet bağlantınızı kontrol edin.';
  static const String errorTooManyRequests =
      'Çok fazla deneme yapıldı. Bir süre bekleyip tekrar deneyin.';
  static const String errorProviderDisabled =
      'Bu giriş yöntemi Firebase Console üzerinde etkin değil.';
  static const String errorSignInFailed = 'Giriş yapılamadı.';
  static const String errorSignOutFailed = 'Oturum kapatılamadı.';
  static const String errorPermissionDenied = 'Bu işlem için yetkiniz yok.';
  static const String errorSaveFailed = 'Kaydetme işlemi tamamlanamadı.';
  static const String errorDeleteFailed = 'Silme işlemi tamamlanamadı.';
  static const String errorHistoryLoadFailed = 'Konum geçmişi okunamadı.';
  static const String errorLocationWriteFailed =
      'Konum sunucuya yazılamadı.';
  static const String errorVehiclesStream =
      'Araç listesi alınamadı. Bağlantınızı kontrol edin.';
  static const String errorGroupConfigsStream = 'Grup ayarları alınamadı.';
  static const String errorAdminStream = 'Yönetici yetkisi okunamadı.';

  // ------------------------------------------------------- Geçici içerikler
  static const String screenComingSoon = 'Bu ekran sonraki aşamada eklenecek.';
  static const String mapScreenTitle = 'Harita';
  static const String registrationScreenTitle = 'Araç Kaydı';
  static const String adminScreenTitle = 'Yönetim';
  static const String settingsScreenTitle = 'Ayarlar';

  // ---------------------------------------------------------------- Zaman
  static const String timeJustNow = 'Az önce';
  static const String timeMinutesAgo = 'dakika önce';
  static const String timeHoursAgo = 'saat önce';
  static const String timeDaysAgo = 'gün önce';
  static const String timeNever = 'Hiç güncellenmedi';
  static const String unitSpeed = 'km/s';

  // -------------------------------------------------------------- Harita
  static const String mapSummaryTitle = 'Filo özeti';
  static const String mapSummaryTotal = 'Toplam';
  static const String mapStatusMoving = 'Hareketli';
  static const String mapStatusStopped = 'Duruyor';
  static const String mapStatusOffline = 'Çevrimdışı';
  static const String mapStatusPending = 'Onay bekliyor';
  static const String mapFilterAll = 'Tümü';
  static const String mapFilterUngrouped = 'Grupsuz';
  static const String mapEmptyTitle = 'Görüntülenecek araç yok';
  static const String mapEmptyDriverHint =
      'Aracınız onaylanıp bir gruba atandığında grubunuzdaki araçlar burada '
      'görünür.';
  static const String mapEmptyAdminHint =
      'Henüz kayıtlı araç yok. Sürücüler Kayıt sekmesinden talep gönderdiğinde '
      'burada görünürler.';
  static const String mapOwnVehicle = 'Bu cihaz';
  static const String mapFieldHidden = 'Gizli';
  static const String mapFieldSpeed = 'Hız';
  static const String mapFieldDriver = 'Sürücü';
  static const String mapFieldGroup = 'Grup';
  static const String mapFieldUpdated = 'Son güncelleme';
  static const String mapGroupNone = 'Atanmamış';
  static const String mapNoLocationYet = 'Bu aracın henüz konumu yok.';
  static const String mapGoToOwnVehicle = 'Aracıma git';
  static const String mapOwnVehicleMissing = 'Bu cihazın haritada konumu yok.';
  static const String mapExpand = 'Genişlet';
  static const String mapCollapse = 'Daralt';
  static const String mapAttribution = 'OpenStreetMap katkıcıları';

  // --------------------------------------------------------------- Kayıt
  static const String registrationNoVehicleTitle =
      'Bu cihaz için araç kaydı yok';
  static const String registrationNoVehicleHint =
      'Plaka ve sürücü adını girip kayıt talebi gönderin. Yönetici '
      'onayladıktan sonra aracınız gruptaki diğer sürücülere görünür.';
  static const String registrationPlateLabel = 'Plaka';
  static const String registrationPlateHint = '40 ABC 123';
  static const String registrationPlateRequired = 'Plaka girin.';
  static const String registrationPlateTooShort =
      'Plaka en az 6 karakter olmalı.';
  static const String registrationNameLabel = 'Sürücü adı';
  static const String registrationNameHint = 'Ad Soyad';
  static const String registrationNameRequired = 'Sürücü adı girin.';
  static const String registrationNameTooShort =
      'Sürücü adı en az 3 karakter olmalı.';
  static const String registrationSubmitNew = 'Kayıt talebi gönder';
  static const String registrationSubmitUpdate = 'Bilgileri güncelle';
  static const String registrationSaved = 'Bilgiler kaydedildi.';
  static const String registrationRequestSent =
      'Kayıt talebiniz gönderildi. Yönetici onayı bekleniyor.';
  static const String registrationStatusApproved = 'Onaylı';
  static const String registrationStatusPending = 'Onay bekliyor';
  static const String registrationApprovedHint =
      'Aracınız grubunuzdaki sürücülerin haritasında görünüyor.';
  static const String registrationPendingHint =
      'Yönetici onaylayana kadar aracınızı haritada yalnızca siz görürsünüz.';
  static const String registrationGroupLabel = 'Grup';
  static const String registrationGroupNone = 'Atanmamış';
  static const String registrationManagedByAdmin =
      'Onay durumunu ve grubu yalnızca yönetici değiştirebilir.';
  static const String registrationAdminSessionTitle = 'Yönetici oturumu açık';
  static const String registrationAdminSessionHint =
      'Araç kaydı sürücü (anonim) oturumuna aittir. Yönetici cihazı takip '
      'edilen bir araç olmamalıdır, bu yüzden kayıt formu kapalıdır. Sürücü '
      'olarak kaydolmak için Ayarlar sekmesinden yönetici oturumunu kapatın.';

  // ------------------------------------------------------------- Yönetim
  static const String adminLoginTitle = 'Yönetici girişi';
  static const String adminLoginHint =
      'Yönetici hesabı Firebase Console üzerinden oluşturulur ve yetkisi '
      'veritabanındaki admins listesinden okunur.';
  static const String adminEmailLabel = 'E-posta';
  static const String adminPasswordLabel = 'Şifre';
  static const String adminEmailRequired = 'E-posta girin.';
  static const String adminEmailInvalid = 'Geçerli bir e-posta girin.';
  static const String adminPasswordRequired = 'Şifre girin.';
  static const String adminLoginButton = 'Giriş yap';
  static const String adminLogoutButton = 'Çıkış yap';
  static const String adminVerifying = 'Yetki doğrulanıyor…';
  static const String adminNotAuthorized =
      'Bu hesap yönetici değil. Oturum kapatıldı.';
  static const String adminNotAuthorizedTitle = 'Yönetici yetkiniz yok';
  static const String adminNotAuthorizedHint =
      'Bu hesap veritabanındaki admins listesinde bulunmuyor. Çıkış yapıp '
      'sürücü oturumuna dönebilirsiniz.';

  static const String adminTabVehicles = 'Araçlar';
  static const String adminTabGroups = 'Gruplar';

  static const String adminSummaryTotal = 'Toplam';
  static const String adminSummaryPending = 'Bekleyen';
  static const String adminSummaryApproved = 'Onaylı';
  static const String adminSummaryGroups = 'Grup';

  static const String adminNoVehicles = 'Kayıtlı araç yok';
  static const String adminNoVehiclesHint =
      'Sürücüler Kayıt sekmesinden talep gönderdiğinde burada görünürler.';
  static const String adminApprove = 'Onayla';
  static const String adminRevoke = 'Onayı kaldır';
  static const String adminAssignGroup = 'Grup ata';
  static const String adminViewHistory = 'Konum geçmişi';
  static const String adminDeleteVehicle = 'Aracı sil';
  static const String adminDeleteVehicleTitle = 'Araç silinsin mi?';
  static const String adminDeleteVehicleMessage =
      'Araç kaydı ve tüm konum geçmişi kalıcı olarak silinecek. Bu işlem geri '
      'alınamaz.';
  static const String adminDelete = 'Sil';
  static const String adminSave = 'Kaydet';
  static const String adminVehicleApproved = 'Araç onaylandı.';
  static const String adminVehicleRevoked = 'Aracın onayı kaldırıldı.';
  static const String adminVehicleDeleted = 'Araç ve konum geçmişi silindi.';
  static const String adminGroupAssigned = 'Grup ataması güncellendi.';
  static const String adminGroupNone = 'Grupsuz';
  static const String adminSelectGroupTitle = 'Grup seç';
  static const String adminNoGroups = 'Henüz grup yok';
  static const String adminNoGroupsHint =
      'Gruplar sekmesinden yeni bir grup oluşturun.';

  static const String adminCreateGroup = 'Grup oluştur';
  static const String adminGroupNameLabel = 'Grup adı';
  static const String adminGroupNameHint = 'Merkez';
  static const String adminGroupNameRequired = 'Grup adı girin.';
  static const String adminGroupNameExists = 'Bu adda bir grup zaten var.';
  static const String adminGroupCreated = 'Grup oluşturuldu.';
  static const String adminEditGroup = 'Grup ayarları';
  static const String adminVisibleGroupsLabel = 'Görünür gruplar';
  static const String adminVisibleGroupsHint = 'Merkez, Kaman';
  static const String adminVisibleGroupsHelp =
      'Bu gruptaki sürücüler kendi grubuna ek olarak burada yazan grupları '
      'görür. Virgülle ayırın.';
  static const String adminShowSpeed = 'Hız görünsün';
  static const String adminShowDriverName = 'Sürücü adı görünsün';
  static const String adminFieldVisibilityHelp =
      'Kapatılan alan bu gruptaki araçlarda diğer sürücülere gizlenir. '
      'Sürücü kendi aracını her zaman tam görür.';
  static const String adminGroupSaved = 'Grup ayarları kaydedildi.';
  static const String adminDeleteGroupTitle = 'Grup silinsin mi?';
  static const String adminDeleteGroupMessage =
      'Bu gruptaki araçların grup ataması kaldırılacak ve diğer grupların '
      'görünür listelerinden çıkarılacak.';
  static const String adminGroupDeleted = 'Grup silindi.';
  static const String adminVehicleCountSuffix = 'araç';

  // ------------------------------------------------------- Konum geçmişi
  static const String historyTitle = 'Konum geçmişi';
  static const String historyEmpty = 'Bu araç için konum kaydı yok.';
  static const String historyRefresh = 'Yenile';
  static const String historyRecordSuffix = 'kayıt';
  static const String historyLimitNote =
      'En yeni 500 kayıt gösterilir, en yeni üstte.';

  // ------------------------------------------------------- Konum servisi
  static const String serviceChannelName = 'Konum takibi';
  static const String serviceChannelDescription =
      'Araç konumunun sunucuya gönderilebilmesi için kalıcı bildirim.';
  static const String serviceNotificationTitle = 'Filo Takip çalışıyor';
  static const String serviceNotificationText =
      'Aracınızın konumu gönderiliyor.';

  static const String trackingSection = 'Konum takibi';
  static const String trackingSwitch = 'Arka planda konum gönder';
  static const String trackingSwitchHint =
      'Açıkken uygulama kapalı olsa da aracınızın konumu gönderilir.';
  static const String trackingPermissionLabel = 'Konum izni';
  static const String trackingStateOn = 'Servis çalışıyor';
  static const String trackingStateOff = 'Servis kapalı';

  static const String permissionServiceDisabled =
      'Cihazın konum servisi kapalı';
  static const String permissionDenied = 'İzin verilmedi';
  static const String permissionForegroundOnly =
      'Yalnızca uygulama açıkken';
  static const String permissionAlways = 'Her zaman (arka plan dahil)';
  static const String permissionDeniedForever =
      'İzin kalıcı olarak reddedildi';

  static const String permissionServiceDisabledTitle = 'Konum servisi kapalı';
  static const String permissionServiceDisabledRationale =
      'Cihazın konum servisi kapalı olduğu için konum alınamıyor. Konum '
      'ayarlarından açmanız gerekiyor.';
  static const String trackingPermissionAction = 'İzin ver';
  static const String permissionForegroundTitle = 'Konum izni gerekli';
  static const String permissionForegroundRationale =
      'Aracınızın haritada görünebilmesi için konum izni vermeniz gerekiyor. '
      'İzin vermezseniz harita çalışmaya devam eder, yalnızca kendi konumunuz '
      'gönderilmez.';
  static const String permissionBackgroundTitle = 'Arka plan konum izni';
  static const String permissionBackgroundRationale =
      'Uygulama kapalıyken de konum gönderebilmek için konum iznini '
      '"Her zaman izin ver" yapmanız gerekiyor. Android bu seçeneği yalnızca '
      'uygulama ayarları ekranından sunar.';
  static const String permissionDeniedForeverRationale =
      'Konum izni kalıcı olarak reddedilmiş. Devam etmek için uygulama '
      'ayarlarından izni elle vermeniz gerekiyor.';
  static const String permissionOpenAppSettings = 'Uygulama ayarlarını aç';
  static const String permissionOpenLocationSettings = 'Konum ayarlarını aç';
  static const String permissionContinue = 'Devam et';
  static const String permissionSkip = 'Şimdilik geç';

  static const String trackingNeedsVehicle =
      'Önce Kayıt sekmesinden araç kaydı oluşturun.';
  static const String trackingNeedsPermission =
      'Konum izni verilmediği için takip başlatılamadı.';
  static const String trackingNeedsNotification =
      'Bildirim izni verilmedi. Kalıcı bildirim olmadan servis çalışamaz.';
  static const String trackingStarted = 'Konum gönderimi başladı.';
  static const String trackingStopped = 'Konum gönderimi durduruldu.';
  static const String trackingStartFailed = 'Konum servisi başlatılamadı.';
  static const String trackingForegroundOnlyWarning =
      'Takip başladı. İzin "yalnızca uygulama açıkken" olduğu için konum, '
      'kalıcı bildirim göründüğü sürece gönderilir; servis kapanırsa '
      'kendiliğinden devam edemez. Kesintisiz takip için "Her zaman izin ver" '
      'önerilir.';
  static const String trackingSimulationNote =
      'Simülasyon kipinde konum sunucuya yazılmaz; yalnızca uygulama açıkken '
      'sahte veriye işlenir.';
  static const String trackingVehicleDeleted =
      'Araç kaydı silindiği için konum gönderimi durduruldu.';

  // ------------------------------------------------------------- Ayarlar
  static const String settingsSessionSection = 'Oturum';
  static const String settingsSessionUid = 'Oturum kimliği (uid)';
  static const String settingsSessionAnonymous = 'Anonim oturum (sürücü)';
  static const String settingsSessionEmail = 'E-posta oturumu (yönetici)';
  static const String settingsAdminDeviceWarning =
      'Yönetici girişi bu cihazdaki anonim sürücü oturumunun yerini alır. '
      'Bu yüzden yönetici cihazı takip edilen bir araç olmamalıdır.';
}

File: panel/js/strings.js
// Paneldeki tüm kullanıcıya görünen metinler burada toplanır.
//
// Kural, uygulamadaki lib/core/strings.dart ile aynı: görünüm dosyalarının
// içine metin yazılmaz, önce buraya eklenir sonra kullanılır. Uygulamayla
// ortak olan metinler birebir aynı tutulur ki iki arayüz aynı dili konuşsun.

export const Strings = Object.freeze({
  // ---------------------------------------------------------------- Genel
  appTitle: 'Filo Takip',
  panelTitle: 'Filo Takip Yönetim Paneli',
  loading: 'Yükleniyor…',
  retry: 'Tekrar dene',
  cancel: 'Vazgeç',
  close: 'Kapat',
  ok: 'Tamam',
  visible: 'Görünür',
  hidden: 'Gizli',
  save: 'Kaydet',
  search: 'Ara',
  refresh: 'Yenile',

  // ---------------------------------------------------------- Simülasyon
  demoBadge: 'DEMO',
  demoNoticeTitle: 'Demo kipi',
  demoNotice:
    'Panel şu anda sahte verilerle çalışıyor. Hiçbir veri sunucuya yazılmıyor '
    + 've sayfayı yenileyince değişiklikler kaybolur.',
  demoAccountsHint: 'Demo hesapları:',
  demoAdminAccount: 'yonetici@ornek.com / 123456 — yönetici',
  demoViewerAccount: 'izleyici@ornek.com / 123456 — izleyici (Merkez grubu)',
  demoPendingAccount: 'bekleyen@ornek.com / 123456 — onay bekleyen hesap',
  demoConfigHint:
    'Gerçek veriye geçmek için js/config.js dosyasına Firebase web '
    + 'yapılandırmasını yapıştırın.',

  // ------------------------------------------------------------- Gezinme
  navMap: 'Harita',
  navVehicles: 'Araçlar',
  navGroups: 'Gruplar',
  navUsers: 'Kullanıcılar',
  navSettings: 'Ayarlar',

  // --------------------------------------------------------------- Giriş
  loginTitle: 'Panele giriş',
  loginSubtitle: 'Filo Takip yönetim paneli',
  loginEmailLabel: 'E-posta',
  loginPasswordLabel: 'Şifre',
  loginButton: 'Giriş yap',
  loginBusy: 'Giriş yapılıyor…',
  logoutButton: 'Çıkış yap',
  registerTitle: 'Yeni hesap',
  registerNameLabel: 'Ad Soyad',
  registerButton: 'Hesap oluştur',
  registerBusy: 'Hesap oluşturuluyor…',
  registerSwitch: 'Hesabınız yok mu? Kayıt olun',
  loginSwitch: 'Zaten hesabınız var mı? Giriş yapın',
  registerHint:
    'Kaydolduktan sonra hesabınız yönetici onayı bekler. Yönetici sizi bir '
    + 'gruba atayınca o grubun araçlarını haritada görürsünüz.',

  emailRequired: 'E-posta girin.',
  emailInvalid: 'Geçerli bir e-posta girin.',
  passwordRequired: 'Şifre girin.',
  passwordTooShort: 'Şifre en az 6 karakter olmalı.',
  nameRequired: 'Ad Soyad girin.',
  nameTooShort: 'Ad Soyad en az 3 karakter olmalı.',

  // ------------------------------------------------------------- Hatalar
  errorInvalidCredentials: 'E-posta veya şifre hatalı.',
  errorNetwork: 'Ağ bağlantısı kurulamadı. İnternet bağlantınızı kontrol edin.',
  errorTooManyRequests: 'Çok fazla deneme yapıldı. Bir süre bekleyip tekrar deneyin.',
  errorProviderDisabled: 'Bu giriş yöntemi Firebase Console üzerinde etkin değil.',
  errorEmailInUse: 'Bu e-posta ile bir hesap zaten var.',
  errorWeakPassword: 'Şifre çok zayıf. En az 6 karakter kullanın.',
  errorSignInFailed: 'Giriş yapılamadı.',
  errorSignOutFailed: 'Oturum kapatılamadı.',
  errorPermissionDenied: 'Bu işlem için yetkiniz yok.',
  errorSaveFailed: 'Kaydetme işlemi tamamlanamadı.',
  errorDeleteFailed: 'Silme işlemi tamamlanamadı.',
  errorHistoryLoadFailed: 'Konum geçmişi okunamadı.',
  errorVehiclesStream: 'Araç listesi alınamadı. Bağlantınızı kontrol edin.',
  errorGroupConfigsStream: 'Grup ayarları alınamadı.',
  errorUsersStream: 'Kullanıcı listesi alınamadı.',
  errorAdminStream: 'Yönetici yetkisi okunamadı.',
  errorStartupTitle: 'Panel başlatılamadı',
  errorFirebaseInit:
    'Firebase bağlantısı kurulamadı. js/config.js içindeki değerleri kontrol edin.',

  // --------------------------------------------------------------- Roller
  roleAdmin: 'Yönetici',
  roleViewer: 'İzleyici',
  rolePending: 'Onay bekliyor',
  roleAdminHint: 'Tüm araçları görür, her ayarı değiştirebilir.',
  roleViewerHint: 'Yalnızca atandığı grubun araçlarını salt-okunur görür.',
  rolePendingHint: 'Yönetici onaylayana kadar hiçbir araç görünmez.',

  pendingTitle: 'Hesabınız onay bekliyor',
  pendingMessage:
    'Yönetici hesabınızı onaylayıp bir gruba atadığında o grubun araçları '
    + 'haritada görünmeye başlayacak. Bu sayfa kendiliğinden güncellenir.',
  viewerNoGroupTitle: 'Henüz bir gruba atanmadınız',
  viewerNoGroupMessage:
    'Hesabınız onaylı ama bir gruba atanmadığı için görüntülenecek araç yok. '
    + 'Yöneticinin sizi bir gruba atamasını bekleyin.',

  // -------------------------------------------------------------- Harita
  mapSummaryTitle: 'Filo özeti',
  mapSummaryTotal: 'Toplam',
  mapStatusMoving: 'Hareketli',
  mapStatusStopped: 'Duruyor',
  mapStatusOffline: 'Çevrimdışı',
  mapStatusPending: 'Onay bekliyor',
  mapFilterAll: 'Tümü',
  mapFilterUngrouped: 'Grupsuz',
  mapEmptyTitle: 'Görüntülenecek araç yok',
  mapEmptyAdminHint:
    'Henüz kayıtlı araç yok. Sürücüler uygulamadaki Kayıt sekmesinden talep '
    + 'gönderdiğinde burada görünürler.',
  mapEmptyViewerHint:
    'Grubunuzda haritada gösterilecek araç yok.',
  mapFieldHidden: 'Gizli',
  mapFieldSpeed: 'Hız',
  mapFieldDriver: 'Sürücü',
  mapFieldGroup: 'Grup',
  mapFieldUpdated: 'Son güncelleme',
  mapFieldCoords: 'Koordinat',
  mapGroupNone: 'Atanmamış',
  mapNoLocationYet: 'Bu aracın henüz konumu yok.',
  mapFitAll: 'Tümünü sığdır',
  mapLayerStreet: 'Sokak',
  mapLayerSatellite: 'Uydu',
  mapFollow: 'Takip et',
  mapFollowing: 'Takip ediliyor',
  mapHiddenNoLocation: 'konumu olmayan araç haritada gösterilmiyor',

  // ---------------------------------------------------------------- Zaman
  timeJustNow: 'Az önce',
  timeMinutesAgo: 'dakika önce',
  timeHoursAgo: 'saat önce',
  timeDaysAgo: 'gün önce',
  timeNever: 'Hiç güncellenmedi',
  unitSpeed: 'km/s',

  // -------------------------------------------------------------- Araçlar
  vehiclesTitle: 'Araçlar',
  vehiclesSearchHint: 'Plaka veya sürücü ara',
  colPlate: 'Plaka',
  colDriver: 'Sürücü',
  colGroup: 'Grup',
  colStatus: 'Durum',
  colSpeed: 'Hız',
  colUpdated: 'Son güncelleme',
  colActions: 'İşlemler',

  summaryTotal: 'Toplam',
  summaryPending: 'Bekleyen',
  summaryApproved: 'Onaylı',
  summaryGroups: 'Grup',

  noVehicles: 'Kayıtlı araç yok',
  noVehiclesHint:
    'Sürücüler uygulamadaki Kayıt sekmesinden talep gönderdiğinde burada '
    + 'görünürler.',
  noSearchResult: 'Aramayla eşleşen kayıt yok.',

  approve: 'Onayla',
  revoke: 'Onayı kaldır',
  assignGroup: 'Grup ata',
  viewHistory: 'Konum geçmişi',
  showOnMap: 'Haritada göster',
  deleteVehicle: 'Aracı sil',
  deleteVehicleTitle: 'Araç silinsin mi?',
  deleteVehicleMessage:
    'Araç kaydı ve tüm konum geçmişi kalıcı olarak silinecek. Bu işlem geri '
    + 'alınamaz.',
  delete: 'Sil',
  vehicleApproved: 'Araç onaylandı.',
  vehicleRevoked: 'Aracın onayı kaldırıldı.',
  vehicleDeleted: 'Araç ve konum geçmişi silindi.',
  groupAssigned: 'Grup ataması güncellendi.',
  groupNone: 'Grupsuz',
  selectGroupTitle: 'Grup seç',

  // -------------------------------------------------------------- Gruplar
  groupsTitle: 'Gruplar',
  noGroups: 'Henüz grup yok',
  noGroupsHint: 'Yeni bir grup oluşturarak başlayın.',
  createGroup: 'Grup oluştur',
  groupNameLabel: 'Grup adı',
  groupNameHint: 'Merkez',
  groupNameRequired: 'Grup adı girin.',
  groupNameExists: 'Bu adda bir grup zaten var.',
  groupCreated: 'Grup oluşturuldu.',
  editGroup: 'Grup ayarları',
  visibleGroupsLabel: 'Görünür gruplar',
  visibleGroupsHelp:
    'Bu gruptaki sürücüler ve izleyiciler kendi grubuna ek olarak burada '
    + 'işaretlenen grupları görür.',
  showSpeed: 'Hız görünsün',
  showDriverName: 'Sürücü adı görünsün',
  fieldVisibilityHelp:
    'Kapatılan alan bu gruptaki araçlarda diğer sürücülere gizlenir. Sürücü '
    + 'kendi aracını, yönetici ise tüm araçları her zaman tam görür.',
  groupSaved: 'Grup ayarları kaydedildi.',
  deleteGroupTitle: 'Grup silinsin mi?',
  deleteGroupMessage:
    'Bu gruptaki araçların grup ataması kaldırılacak, gruba atanmış '
    + 'izleyiciler grupsuz kalacak ve grup, diğer grupların görünür '
    + 'listelerinden çıkarılacak.',
  groupDeleted: 'Grup silindi.',
  vehicleCountSuffix: 'araç',
  viewerCountSuffix: 'izleyici',

  // --------------------------------------------------------- Kullanıcılar
  usersTitle: 'Panel kullanıcıları',
  usersHint:
    'Panele e-posta ile kaydolan hesaplar burada listelenir. Yetkiyi ve '
    + 'grubu yalnızca yönetici değiştirir.',
  colUser: 'Kullanıcı',
  colEmail: 'E-posta',
  colRole: 'Yetki',
  colCreated: 'Kayıt',
  noUsers: 'Panele kayıtlı kullanıcı yok',
  noUsersHint: 'Kullanıcılar giriş ekranındaki "Kayıt olun" bağlantısını kullanır.',
  userApproved: 'Kullanıcı onaylandı.',
  userRevoked: 'Kullanıcının onayı kaldırıldı.',
  userGroupAssigned: 'Kullanıcının grubu güncellendi.',
  userRoleChanged: 'Kullanıcının yetkisi güncellendi.',
  deleteUserTitle: 'Kullanıcı kaydı silinsin mi?',
  deleteUserMessage:
    'Panel kaydı silinecek ve kullanıcı yeniden onay bekleyen duruma düşecek. '
    + 'Firebase Authentication hesabı silinmez; onu Firebase Console üzerinden '
    + 'kaldırmanız gerekir.',
  userDeleted: 'Kullanıcı kaydı silindi.',
  makeAdmin: 'Yönetici yap',
  removeAdmin: 'Yöneticiliği kaldır',
  makeAdminTitle: 'Yönetici yapılsın mı?',
  makeAdminMessage:
    'Bu hesap tüm araçları görebilecek, grupları ve diğer kullanıcıların '
    + 'yetkilerini değiştirebilecek. Yönetici yetkisi sınırsızdır.',
  removeAdminTitle: 'Yöneticilik kaldırılsın mı?',
  removeAdminMessage:
    'Hesap yönetici listesinden çıkarılacak ve izleyici yetkisine düşecek.',
  selfRoleLocked: 'Kendi yetkinizi bu ekrandan değiştiremezsiniz.',
  youBadge: 'Siz',

  // ------------------------------------------------------- Geçmiş rota
  playbackTitle: 'Geçmiş rota',
  playbackLoading: 'Geçmiş yükleniyor…',
  playbackEmpty: 'Bu araç için konum kaydı yok.',
  playbackNoRoute: 'Kayıtlar tek noktada; çizilecek rota yok.',
  playbackPlay: 'Oynat',
  playbackPause: 'Duraklat',
  playbackReplay: 'Baştan',
  playbackSpeedLabel: 'Hız',
  playbackSpeedSlow: 'Yavaş',
  playbackSpeedNormal: 'Normal',
  playbackSpeedFast: 'Hızlı',
  playbackBackToLive: 'Canlıya dön',
  playbackShowRecords: 'Kayıtları göster',
  playbackFollow: 'Haritada takip et',
  stopLabel: 'Durak',

  reportDistance: 'Katedilen yol',
  reportMoving: 'Hareket süresi',
  reportStopped: 'Duruş süresi',
  reportMaxSpeed: 'En yüksek hız',
  reportAvgSpeed: 'Ortalama hız',
  reportStops: 'Durak sayısı',
  reportRange: 'Kapsanan aralık',
  reportWindowNote:
    'Uygulama araç başına en fazla 500 konum kaydı tutar (45 saniyede bir), '
    + 'yani geçmiş yaklaşık son 6 saati kapsar. Daha eskisi otomatik silinir.',

  // ------------------------------------------------------- Konum geçmişi
  historyTitle: 'Konum geçmişi',
  historyEmpty: 'Bu araç için konum kaydı yok.',
  historyRecordSuffix: 'kayıt',
  historyLimitNote: 'En yeni 500 kayıt gösterilir, en yeni üstte.',
  historyShowTrack: 'Rotayı haritada göster',
  historyHideTrack: 'Rotayı gizle',
  historyTrackShown: 'Rota haritada çizildi.',
  colTime: 'Zaman',
  colCoords: 'Koordinat',

  // ------------------------------------------------------------- Ayarlar
  settingsTitle: 'Ayarlar',
  settingsSession: 'Oturum',
  settingsAccount: 'Hesap',
  settingsUid: 'Oturum kimliği (uid)',
  settingsEmail: 'E-posta',
  settingsRole: 'Yetki',
  settingsGroup: 'Grup',
  settingsBackend: 'Arka uç',
  settingsBackendDemo: 'Demo (bellek içi sahte veri)',
  settingsBackendFirebase: 'Firebase Realtime Database',
  settingsProject: 'Proje',
  settingsAbout: 'Panel hakkında',
  settingsAboutText:
    'Bu panel, Filo Takip Android uygulamasıyla aynı veritabanını okur. '
    + 'Araç kaydı ve konum gönderimi yalnızca uygulamadan yapılır; panel '
    + 'yönetim ve izleme içindir.',
});
